<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SuRaj" failures="0" tests="1" name="test.DataProvider" time="0.000" errors="1" timestamp="2023-01-19T12:55:24 IST" skipped="0">
  <testcase classname="test.DataProvider" name="TestData" time="0.000">
    <error message="
Cannot instantiate class pom.DateProvider" type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Cannot instantiate class pom.DateProvider
at org.testng.internal.InstanceCreator.newInstance(InstanceCreator.java:36)
at org.testng.internal.objects.SimpleObjectDispenser.dispense(SimpleObjectDispenser.java:40)
at org.testng.internal.objects.GuiceBasedObjectDispenser.dispense(GuiceBasedObjectDispenser.java:30)
at org.testng.internal.Parameters.findDataProvider(Parameters.java:651)
at org.testng.internal.Parameters.findDataProvider(Parameters.java:553)
at org.testng.internal.Parameters.handleParameters(Parameters.java:772)
at org.testng.internal.Parameters.handleParameters(Parameters.java:740)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:59)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:38)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:791)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.InstantiationException: pom.DateProvider
at java.base/java.lang.Class.newInstance(Class.java:671)
at org.testng.internal.InstanceCreator.newInstance(InstanceCreator.java:30)
... 15 more
Caused by: java.lang.NoSuchMethodException: pom.DateProvider.<init>()
at java.base/java.lang.Class.getConstructor0(Class.java:3617)
at java.base/java.lang.Class.newInstance(Class.java:658)
... 16 more
]]>
    </error>
  </testcase> <!-- TestData -->
  <system-out/>
</testsuite> <!-- test.DataProvider -->
